spec/index.html

```diff
@@ -144,21 +152,55 @@ <h3>Simple Triples</h3>
         (<a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>,
         <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>, and
         <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>) terms,
-        and terminated by '<code>.</code>'.
+        and terminated by '<code>.</code>',
+        which describes a <a data-cite="RDF12-CONCEPTS#dfn-simple-triple">simple triple</a>.
         White space (spaces <code>U+0020</code> or tabs <code>U+0009</code>) may surround terms,
         except where significant as noted in the <a href="#n-triples-grammar">grammar</a>.</p>
 
       <p>Comments are treated as white space, and may be given after a '<code>#</code>' that is not part of
         another lexical token and continue to the end of the line.</p>
 
-      <pre id="ex-simple-triple" class="example ntriples" data-transform="updateExample"
+      <p>Simple triples restrict <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>
+        and <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>
+        from being a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>.</p>
+
+      <pre id="ex-simple-triple" class="example N-Triples" data-transform="updateExample"
            title="Simple Triple">
         <!--
         <http://example.org/#spiderman> <http://www.perceive.net/schemas/relationship/enemyOf> <http://example.org/#green-goblin> .
         -->
       </pre>
     </section>
 
+    <section id="complex-triples">
+      <h3>Complex Triples</h3>
+
+      <p>A <a data-cite="RDF12-CONCEPTS#dfn-complex-triple">complex triple</a> statement
+        extends a simple triple statement
+        to allow <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>
+        or <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>
+        to be a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>.</p>
+
+      <p>A <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>
+        is represented as a <code><a href="#grammar-production-quotedTriple">quotedTriple</a></code> with
+        <code><a href="#grammar-production-subject">subject</a></code>,
+        <code><a href="#grammar-production-predicate">predicate</a></code>, and
+        <code><a href="#grammar-production-object">object</a></code>
+        proceeded by `&lt;&lt;` and followed by `>>`.
+        Note that <a data-cite="RDF12-CONCEPTS#dfn-complex-triple">complex triples</a>
+        may be recursive.
+       </p>
+
+      <pre id="ex-complex-triple" class="example complex N-Triples" data-transform="updateExample"
+           title="Complex Triple">
+        <!--
+        <http://example.org/employee38> <http://example.org/familyName> "Smith" .
+        << <http://example.org/jobTitle> <http://example.org/jobTitle> "Assistant Designer" >>
+           <http://example.org/accordingTo> <http://example.org/employee22> .
```

Comment:
As elsewhere...
```suggestion
PREFIX  ex:  <http://example.org/>

ex:employee38                                         ex:familyName   "Smith" .
<< ex:jobTitle  ex:jobTitle  "Assistant Designer" >>  ex:accordingTo  ex:employee22 .
```

Reply:
We can't use prefixes in N-Triples. It was either make a long line to keep the triple to a single line, or wrap to another line.

Reply:
Ugh. Right. OK, given that we're making an illegal triple with one line break, I suggest some more, either like this --
```suggestion
<http://example.org/employee38>
    <http://example.org/familyName>
        "Smith" .
<< <http://example.org/jobTitle> <http://example.org/jobTitle> "Assistant Designer" >>
    <http://example.org/accordingTo>
        <http://example.org/employee22> .
```

Reply:
Line breaks aren't illegal in N-Triples, as they're just whitespace. It's only in the canonical form that line breaks (or anything other than proscribed white space) is illegal.

Reply:
I added Typographical Conventions which say that newlines in examples are treated as horizontal white space unless represented using &#x2424;. This also adds exemplary white space to other examples to help improve readability.

Reply:
It's syntactically illegal so I don't think that improves explanation.  The user can't use the example text as given.

N-triples lines can be long. We have to accept that - formatting them as bad syntax is worse IMO.

The user can't use the example text as given.

Reply:
How would you feel about using still valid, but different and simplified IRIs in these examples. For example:

```turtle
<ex:e38> <ex:fn> "Smith" .
<< <ex:38> <ex:title> "Designer" >> <ex:says> <ex:e22> .
```

Reply:
I'll make the change, but note that I've updated to use blank nodes for a more compact representation that sticks to http IRIs.